Modules/CLI/FiberTractMeasurements/Testing/CompareTxts.cxx

```diff
@@ -26,36 +50,83 @@ int main( int argc, char * argv[] )
   std::ifstream resultTxt(resultTxtPath.c_str());
   std::ifstream baselineTxt(baselineTxtPath.c_str());
 
-  std::string resultLine;
-  std::string baselineLine;
-  std::string resultMeasure;
-  std::string baselineMeasure;
-  int c = 1;
-  while (std::getline(resultTxt, resultLine) && std::getline(baselineTxt, baselineLine))
-   {
-      if (c > 1)
+  auto fileExists = doesFileExist(baselineTxtPath);
```

Comment:
Since the path of the file may not be self descriptive, the functions should probably be more informative and indicate that the baseline is missing.

Reply:
The method `doesFileExist` is used to test both the baseline and the result, so adding any additional string to tell whether it is the baseline or the result adds more code. Ideally, this could be avoided if we could use the CMake comparison commands. For the time being, given that that requires further changes (https://github.com/SlicerDMRI/SlicerDMRI/pull/211#issuecomment-1806644986), I prefer to keep this as they are.